src/theory/quantifiers/ematching/trigger_term_info.h

```diff
@@ -108,6 +108,19 @@ class TriggerTermInfo
   static bool isRelationalTrigger(Node n);
   /** Is k a relational trigger kind? */
   static bool isRelationalTriggerKind(Kind k);
+  /**
+   * Is n a usable relational trigger, which is true if RelationalMatchGen
+   * can process n.
+   */
+  static bool isUsableRelationTrigger(Node n);
+  /**
+   * Same as above, but lit / hasPol / pol are updated to the required
+   * constructor arguments for RelationalMatchGen.
```

Comment:
```suggestion
   * Same as above, but lit / hasPol / pol are updated to the required
   * constructor arguments for RelationalMatchGenerator.
```
Moreover, what is `lit` here seems to be `rtrigger` in that class, so you should sync them.